CIScripts/Testenv/Testenv.Tests.ps1

```diff
@@ -29,18 +43,23 @@ Describe "Testenv" {
 
         BeforeEach {
             $Yaml = @"
-controller:
-  os_credentials:
-    username: AzureDiamond
-    password: hunter2
-    address: 1.2.3.1
-    port: 5000
+openStack:
+  username: AzureDiamond
+  password: hunter2
+  project: admin
+  address: 1.2.3.1
+  port: 5000
 
-  rest_api:
-    address: 1.2.3.1
-    port: 8082
+controller:
+  address: 1.2.3.1
+  restApiPort: 8082
+  defaultProject: ci_tests
 
-  default_project: ci_tests
+testbed:
+  AdapterName: Eth1
+  VHostName: vEth
+  ForwardingExtensionName: MyExtension
+  AgentConfigFilePath: MyFile
 
 testbeds:
```

Comment:
Maybe we should change this name to `hosts` or something because these two sections describe completely different vars and we should seperate them by using different names

Reply:
But we can do this in another PR as we are waiting for this PR to be merged

Reply:
@m-kostrzewa Do you want to join the bikeshedding? :)